### PR TITLE
Add model fitting benchmarks

### PR DESCRIPTION
**Pull request by WilliamJamieson**
This adds some benchmarks involving the model fitting in Astropy.

### CODE REVIEW

File: benchmarks/modeling/fitting.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+Gaussian2D = models.Gaussian2D(amplitude=1, x_mean=0,  y_mean=0, \
+        x_stddev=1, y_stddev=1)
+
+Polynomial1D = models.Polynomial1D(degree=2)
+Polynomial2D = models.Polynomial2D(degree=2)
+
+Chebyshev1D = models.Chebyshev1D(degree=2)
+Chebyshev2D = models.Chebyshev2D(x_degree=2, y_degree=2)
+
+combined_gauss_1d = models.Gaussian1D(1, 0, 0.1) + models.Gaussian1D(2, 0.5, 0.1)
```
Comment (nden) [inline benchmarks/modeling/fitting.py:17]: Let's have one test with a larger number of models, e.g. 20, instead of 2.

Comment (WilliamJamieson) [reply benchmarks/modeling/fitting.py:17]: How realistic should the parameters of the models being combined be?

**Comment by pllim** [comment]
Benchmarks are currently not running... Solution TBA but timeframe is unknown. FYI.

**Review by nden — APPROVED** [review]

**Comment by nden** [comment]
@pllim Thanks for the heads up. These run locally with scipy 1.6. I am going to merge it so it's ready for better times.

**Comment by nden** [comment]
I guess I'm not going to merge it :-)

**Comment by pllim** [comment]
Is all the random arrays necessary? That would introduce ambiguity to the results, no?

**Comment by pllim** [comment]
Is this still necessary or has this been incorporate into STScI fork?

**Comment by pllim** [comment]
Superseded by #110 , I think.

**Pull request closed without merging**